### DISCUSSION TITLE
2 short questions

### DISCUSSION THREAD

**Discussion by AndrewRCD** [rrirower/DP2SceneGraphWizard#91] [Q&A]
1. Can the video thumbnail image be added to the video page?  when we have only the long description now - 
2.  in the Wiki for Content Voting you say Edit the ContentVotingTask.brs file and change the value of the 'EndpointUrl' variable to point to the URL of your back-end server that will process the content vote. You MUST change this value. - any documentation of help with how to set up the server part for this? 
thanks

**Comment by rrirower** [comment]
The Wiki contains info on the [optional manifest attributes](https://github.com/rrirower/DP2SceneGraphWizard/wiki/Custom-Manifest-Attributes) that control the appearance of the channel.  In particular, _HIDE_GRID_THUMBNAIL_ can be used to display the content thumbnail in the upper right of the channel.

Content voting is supported by a back-end server.  The server must be set up to use a Rest API.  For testing purposes, I used the [Amazon API gateway](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-rest-api.html) and [Lambda functions](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html) to define an endpointurl that receives the data from the channel.  The lambda function was written in Python.

**Comment by AndrewRCD** [comment]
thanks 
Yes, I did find the one for the primary grid - I was referring to the "individual" video page -  the one you get after you click on a thumbnail in a category -  I do have the thumbnail on the main channel page - where the categories are listed - but if ii click to see a video - o the next page is only text - this is where I was asking about adding the thumbnail image.
If you don't have a long description to use, the page is .......  empty - only two buttons and the video title - and even if you add the long description - still just a lot of text - not really appealing from the design point of view.

**Reply by rrirower** [reply]
Can you post an image so I can see what you're referring to?

**Reply by AndrewRCD** [reply]
of course - thank you for your time
![viideo page](https://github.com/rrirower/DP2SceneGraphWizard/assets/161251722/25c84c4c-ff3b-4757-81bf-f17ac83cbb98)

**Reply by rrirower** [reply]
Is that an episode of a series?

**Reply by AndrewRCD** [reply]
short form -  and it is the same for movies and tv specials.

**Reply by AndrewRCD** [reply]
For series and episode, it populates nicely with the videos from the series.

**Reply by rrirower** [reply]
This project adheres to the Roku Direct Publisher JSON spec.  As such, 'long description' is required.  As an alternative, have you tried using the optional manifest attribute USE_THUMBNAIL_BACKGROUND?

I'll mark this as a future enhancement.

**Comment by rrirower** [comment]
A new custom manifest attribute was added for this in 1.0.0.2.
